rotkehlchen/data/importer.py

```diff
@@ -871,6 +871,24 @@ def _consume_blockfi_entry(self, csv_row: Dict[str, Any], **kwargs: Any) -> None
                 notes=f'{entry_type} from BlockFi',
             )
             self.db_ledger.add_ledger_action(action)
+        elif entry_type == 'Crypto Transfer':
+            category = (
+                AssetMovementCategory.WITHDRAWAL if FVal(csv_row['Amount']) < ZERO
```

Comment:
`amount` is a deserialized version of `csv_row['Amount']` but it returns the absolute value. This means it'll always return a positive amount.

Which is why I did `FVal(csv_row['Amount'])` because any deserialization error is caught earlier in the code.

Reference: https://github.com/rotki/rotki/blob/develop/rotkehlchen/data/importer.py#L812